src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -96,19 +98,48 @@ class ErgoWalletActor(settings: ErgoSettings,
     }
   }
 
+  def scanBlock(block: ErgoFullBlock): Unit = {
```

Comment:
Please add ScalaDoc in particular describing the effects of this method on the Wallet state.

Reply:
done